### DISCUSSION TITLE
Welcome to mapwidget Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/mapwidget#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lalligagger** [comment]
Hello! Excited to see the start of this project. I'd be very interested in a common python API across the popular JS map options. Most of my work is in planning satellite remote sensing missions and occasionally working with the data itself. Would be interested in helping out with especially the Cesium features once I have a better idea of how to implement those here.

**Reply by giswqs** [reply]
You can see the source how it is implemented. Contributions are always welcome.

**Reply by giswqs** [reply]
@lalligagger I have invited you to be collaborator for this project. You are welcome to explore and contribute. Let me know if you have any questions.
